src/ssvc/decision_points/public_safety_impact.py

```diff
@@ -24,42 +27,29 @@
 
 MATERIAL = SsvcDecisionPointValue(
     name="Material",
-    description="""(Any one or more of these conditions hold.) 
-
-*Physical harm*: Does one or more of the following:
-
-- Causes physical distress or injury to system users.
-- Introduces occupational safety hazards.
-- Reduces and/or results in failure of cyber-physical system safety margins.
-
-*Environment*: Major externalities (property damage, environmental damage, etc.) are
-imposed on other parties. 
-
-*Financial*: Financial losses likely lead to bankruptcy of multiple persons. 
- 
-*Psychological*: Widespread emotional or psychological harm, sufficient to necessitate
-counseling or therapy, impact populations of people. 
-""",
+    description="Any one or more of these conditions hold. "
```

Comment:
Fixed formatting enough to make it look ok in markdown. Might still need attention in #380